### DISCUSSION TITLE
Anonymous Voting (only visible to creator)

### DISCUSSION THREAD

**Discussion by christophhillisch** [lukevella/rallly#144] [Ideas]
I would love to hold my survey anonymously, so that only I as the creator of the survey can see who voted for which date.

**Comment by blazekjan** [comment]
Is there any ETA on this @lukevella ?

**Comment by blazekjan** [comment]
Bump. Can we have some info?

**Reply by lukevella** [reply]
It will be part of a bigger update which I am actively working on. Sorry I can't give you an ETA.

**Reply by blazekjan** [reply]
Good to hear that! Thank you.

**Comment by blazekjan** [comment]
Still on the roadmap @lukevella ? Until this is done, we can't implement it for our usage.

**Reply by lukevella** [reply]
Yes it is, aiming for an April release date.

**Reply by blazekjan** [reply]
> Yes it is, aiming for an April release date.

Good to hear that, thank you!

**Reply by daviiiiiid** [reply]
Hi @lukevella, was this released? I've looked around and can't find it in the options. Thanks! It's the only thing missing for me as well for my project.
